Changes.md

```diff
@@ -1,16 +1,24 @@
 0.58.x.x (relative to 0.58.2.0)
 ========
 
+Improvements
+------------
+
+- Preferences : Added support for OpenColorIO context variables.
```

Comment:
Sorry, this is the only place I could think to hang it. It'd be good to note _somewhere_ that the global UI display transform doesn't support dynamic OCIO Contexts.

Reply:
Added in c20b1a2 (and fixed mac build in a942990).